pkg/cmd/cli/backup/describe.go

```diff
@@ -73,7 +73,7 @@ func NewDescribeCommand(f client.Factory, use string) *cobra.Command {
 			}
 
 			first := true
-			for _, backup := range backups.Items {
+			for i, backup := range backups.Items {
```

Comment:
It's unbelievable lint reports an error for this.

Reply:
Technically speaking, this indeed causes problem if we pass the loop variable point to a closure of any ansync operation, so I kept this rule and made the change in code.